src/tests/PhysicsTest.cpp

```diff
@@ -97,9 +98,6 @@ class PhysicsManagerTest : public testing::Test {
     return objectWrapper;
   }
 
-  // must declare these in this order due to avoid deallocation errors
-  esp::gfx::WindowlessContext::uptr context_;
```

Comment:
See my previous comment.

Reply:
Yep, let's resolve the discussion about `context_` here: https://github.com/facebookresearch/habitat-sim/pull/1308/files#r658137220